prac_06/guitar.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Guitar class
+"""
+CURRENT_YEAR = 2017
```

Comment:
Why? Just because my solutions were made in 2017 doesn't mean you made this then 🤔